src/pages/VenuePage/VenuePage.tsx

```diff
@@ -257,8 +253,8 @@ const VenuePage = () => {
     return <LoadingPage />;
   }
 
-  const hasEntrance = isTruthy(venue?.entrance);
   const hasEntered = profile?.enteredVenueIds?.includes(venueId);
+  const hasEntrance = isTruthy(venue?.entrance);
```

Comment:
nit: this was in the other order because that's the order of the boolean expression on line 262~258. Any reason it reversed order?

Reply:
It was a conflic resolution, I will fix it.

Reply:
This one still hasn't been fixed, can we please fix?